doc/conf.py

```diff
@@ -155,6 +155,15 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static', '_images']
 
+# adapted from: http://rackerlabs.github.io/docs-rackspace/tools/rtd-tables.html
+html_context = {
+    'css_files': [
+        '_static/theme_overrides.css',  # override wide tables in RTD theme
+        '_static/altair-gallery.css',
+        '_static/altair-plot.css'
```

Comment:
I don't think the altair-gallery and altair-plot CSS need to be listed here; they are added by the associated sphinx directives if needed.

Reply:
The gallery behaved as described here when I didn't add them. 
https://github.com/altair-viz/altair/issues/415#issuecomment-337715118
In essence, until I added those two additional css files, each image in the minigallery took up its own line with a lot of space in between. And it wasn't the particular css in theme_overrides.css that caused the change. Just the existence of that file (even if empty), when added and referenced in conf.py caused the minigallery to display improperly, until I added all three css files to the html_context.

Reply:
Oh, OK. I see that now. This is strange... I wonder why specifying ``html_context`` overrides ``app.add_stylesheet`` in extensions?

Reply:
Ah, looks like a bug: https://github.com/sphinx-doc/sphinx/issues/2442

Reply:
It seems you can replace the ``html_context`` with the following:
```python
def setup(app):
    app.add_stylesheet('theme_overrides.css')
```
would you mind trying that to see if the results are as you expect?

Reply:
Switching to the setup function worked for me. Added to PR. 

Worth noting that on:
https://github.com/rtfd/sphinx_rtd_theme/issues/117
there was some dialog about it only working locally.

My favorite line in that issue is (and I quote), "The workaround that @smartass101 came up with is a good one."

(That thread is generally hard to follow, but my sense is that all agree that anything in theme_overrides.css should have `!important` added to it.)

If you need me to redo this in a branch, let me know.